CMakeLists.txt

```diff
@@ -1,13 +1,17 @@
 cmake_minimum_required(VERSION 3.18.0)
-project(tobot-engine VERSION 1.0.0 LANGUAGES CXX)
+# We need to build a target with name of the project i fear
+project(tobot-core VERSION 1.0.0 LANGUAGES CXX)
```

Comment:
Maybe this is what i forgot yesterday, and i thought to myself: Is it possible to keep the project name as `tobot-engine`? So that when we link `${PROJECT_NAME}` in example `tobot-core`  is automatically linked?

I mean our project should be technically called tobot-engine. If this is somehow more tricky than imagined, I think its also a possibility to rename the project to just `tobot` and to rename the `tobot_core` module to just `tobot`

Reply:
The PROJECT_SOURCE_DIR variable is messed up for some reason if we don't build a target with that name. The only way to do this right now would be to introduce a new library called tobot_engine.

Reply:
Or to rename the core of course

Reply:
hmm lets keep it this way for now.